tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1191,6 +1214,36 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
       return true;
     }
 
+    if (gemm->custom_call_target() == kCublasLtMatmulF8CallTarget &&
+        bias->shape().element_type() == F32 && convert != nullptr) {
```

Comment:
If `convert` is nullptr, then you want to fail the fusion. So this logic should look like:

```C++
if (gemm->custom_call_target() == kCublasLtMatmulF8CallTarget &&
        bias->shape().element_type() == F32) {
  if (convert == nullptr) {
    return false;
  }
  ... // Rest of the logic
}